frontend/src/views/WheresMyGene/components/GeneSearchBar/index.tsx

```diff
@@ -134,8 +134,8 @@ export default function GeneSearchBar({ onGenesChange }: Props): JSX.Element {
   // DEBUG
```

Comment:
Can these be removed or replaced with a todo?

Reply:
Ah good point! Just removed it 👌 Thank you!